### DISCUSSION TITLE
flusher_http输出插件，如何将json转换为jsonArray输出到接口

### DISCUSSION THREAD

**Discussion by Sic9527** [alibaba/ilogtail#1631] [Help] [answered]
flusher_http输出插件，如何将json转换为jsonArray输出到接口

**Comment by Takuka0311** [comment, marked as answer]
类似这样的配置可以吗？https://open.observability.cn/project/ilogtail/plugins-flusher-http/#_top
![image](https://github.com/user-attachments/assets/8ab8dfdf-5afb-4b46-b353-65a28b446500)
